src/libraries/System.Reflection/tests/PointerTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using System.Reflection;
+
+namespace System.Reflection.Tests
+{
+    public class PointerTests
+    {
+        public unsafe struct BitwiseComparable
+        {
+            public int* PublicInt;
+        }
+
+        public unsafe struct MemberwiseComparable
+        {
+            public string ReferenceType;
+            public int* PublicInt;
+        }
+
+        [Fact]
+        public unsafe void BitwiseEquality_AreEqual()
```

Comment:
I assume this test and BitwiseEquality_EqualWithSelf below pass without the changes in this PR.

Reply:
Yes they do. I've added them for completeness and as regression tests. They don't rely on the Pointer class tho, so this file may be the wrong place for them.